content/exchange/artifacts/Windows.Audit.CISCat_Lite.yaml

```diff
@@ -0,0 +1,62 @@
+name: Windows.Audit.CISCat_Lite
+description: |
+  This artifact runs the CISCat-Lite tool on the target machine and uploads the html output on the velociraptor server.
+  
+  CIS-CAT Lite is a free tool from the Center for Internet Security that helps in assessing and improving IT security configurations. 
+  It enables unlimited system scans, provides compliance scores, and offers remediation steps based on CIS Benchmarks. 
+  This tool is useful for organizations seeking to enhance their technology security.
+
+  Downloads your CIS-CAT Lite free from the https://learn.cisecurity.org/cis-cat-lite link. (Registration is required!)
+  After downloading the zip file, upload the file to the velociraptor server as the CISCat_Lite tool. 
+  The artifact has been configured to perform a standard scan of a Windows 10 Enterprise machine. 
+  To select the baseline and profiles, execute the command ".\Assessor-CLI.bat -i"
+type: CLIENT
+
+author: Antonio Blescia (TheThMando)
+
+parameters:
+  - name: BaselinePath
+    default: "./benchmarks/CIS_Microsoft_Windows_10_Enterprise_Benchmark_v2.0.0-xccdf.xml"
+  - name: ProfileName
+    default: "Level 1 (L1) - Corporate/Enterprise Environment (general use)"
+
+tools:
+  - name: CISCat_Lite
+    serve_locally: true
+
+sources:
+  - precondition: SELECT OS From info() where OS = 'windows'
+
+    query: |
+      -- Generate the temp dir
+      LET TmpDir <= tempdir(remove_last=true)
+      -- Upload the CISCat_Lite tool on the host
+      LET Toolzip <= SELECT FullPath
```

Comment:
FullPath is also deprecated now - use OSPath as it will be removed in futute

Reply:
Fixed